sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -83,15 +83,15 @@ export class ChangeFeedResponse<T> {
 export class ClientContext {
     constructor(cosmosClientOptions: CosmosClientOptions, globalEndpointManager: GlobalEndpointManager);
     // (undocumented)
-    batch<T>({ body, path, partitionKey, resourceId, options }: {
```

Comment:
Prettier changes modified the public API signature for cosmos. When rebuilding the project with rush, this file `cosmos.api.md` was updated. I'm not sure if this is expected.